internal/zinc-apiinfo/src/main/scala/sbt/internal/inc/ClassToAPI.scala

```diff
@@ -24,9 +24,9 @@ object ClassToAPI {
   // (api, public inherited classes)
   def process(classes: Seq[Class[_]]): (Seq[api.ClassLike], Set[(Class[_], Class[_])]) =
     {
-      val pkgs = packages(classes).map(p => new api.Package(p))
+      packages(classes).map(p => new api.Package(p))
```

Comment:
If we really don't use return values we should replace `map` and `flatMap` with `foreach` (and remove overhead for creating unused collections).